src/commands/createNewProject/ProjectCreateStep/DotnetProjectCreateStep.ts

```diff
@@ -31,10 +32,21 @@ export class DotnetProjectCreateStep extends ProjectCreateStepBase {
 
         const projectName: string = path.basename(context.projectPath);
         const projName: string = projectName + language === ProjectLanguage.FSharp ? '.fsproj' : '.csproj';
-        await this.confirmOverwriteExisting(context, projName);
 
-        const majorVersion: string = getMajorVersion(version);
         const workerRuntime = nonNullProp(context, 'workerRuntime');
+        // For containerized function apps we need to call func init before intialization as we want the .csproj file to be overwritten with the correct version
+        // currentely the version created by func init is behind the template version
+        if (context.containerizedProject) {
+            let runtime = 'dotnet-isolated';
+            if (workerRuntime.displayInfo.description === 'LTS') {
```

Comment:
I don't recommend using this property to check for isolated vs. non. `LTS` could potentially change, and it's just a display label.

You're probably better off checking if `capabilities` includes `"isolated"` or not.  We defined the `IWorkerRuntime` so just add capabilities to the typings.